app/code/Magento/InventorySourceSelectionApi/Model/GetAvailableSourceItemsBySkusAndSortedSource.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventorySourceSelectionApi\Model;
+
+use Magento\Framework\Api\SearchCriteriaBuilder;
+use Magento\InventoryApi\Api\Data\SourceItemInterface;
+use Magento\InventoryApi\Api\SourceItemRepositoryInterface;
+
+/**
+ * Retrieve source items for a defined set of skus and sorted source codes
+ *
+ * Useful for determining availability and source selection
+ */
+class GetAvailableSourceItemsBySkusAndSortedSource
+{
+    /**
+     * @var SourceItemRepositoryInterface
+     */
+    private $sourceItemRepository;
+
+    /**
+     * @var SearchCriteriaBuilder
+     */
+    private $searchCriteriaBuilder;
+
+    /**
+     * @param SourceItemRepositoryInterface $sourceItemRepository
+     * @param SearchCriteriaBuilder $searchCriteriaBuilder
+     * @SuppressWarnings(PHPMD.LongVariable)
```

Comment:
Variable names don't seem to be long here

Reply:
`$searchCriteriaBuilder` is more than 20 chars and PHPMD complains about it if I remove the suppression.